src/RobotInterface/src/Helper.cpp

```diff
@@ -60,8 +60,9 @@ bool RobotInterface::getFeedbacksRaw(unsigned int maxAttempts, bool getBaseEst)
     bool okRightWrench = false;
 
     bool okBaseEstimation = true;
-    if(getBaseEst)
-        okBaseEstimation = !m_useExternalRobotBase;
+
+    if(m_useExternalRobotBase)
```

Comment:
The ``if`` should not be necessary. If  ``m_useExternalRobotBase`` is false, ``okBaseEstimation`` remains ``true``.

Reply:
if `m_useExernalRobotBase` is `true` we would like to have `okBaseEstimation=false`

Reply:
In fact, I was suggesting to remove only that line 64: https://github.com/robotology/walking-controllers/blob/27d66392e27b4803ce492ba1da31df8efa665aec/src/RobotInterface/src/Helper.cpp#L64

If ``okBaseEstimation=false``, the code in the following line https://github.com/robotology/walking-controllers/blob/27d66392e27b4803ce492ba1da31df8efa665aec/src/RobotInterface/src/Helper.cpp#L65

makes ``okBaseEstimation=false``.

Reply:
Done!